src/app/gui/mainwindow.cpp

```diff
@@ -227,5 +227,6 @@ MainWindow::~MainWindow() {
 
   // Stop stack:
   multi_stack->stop();
+  delete multi_stack;
```

Comment:
As my camera uses the Ethernet port to communicate, me and my team made a plugin based on the existing ones for it. The thing is, we use the Vimba API, and every time ssl-vision is closed, it gives a segmentation fault. By adding this, and the deletes in `multivisionstack.cpp`, we found that the error was fixed.